doc/source/multivisit_products.rst

```diff
@@ -0,0 +1,243 @@
+addition, most of a typical SkyCell layer will be empty due to the small size of most detectors, especially the WFC3/IR,
+ACS/HRC and ACS/SBC detectors which are only 1024x1024 arrays.  As a result, each SkyCell is evaluated to define only a common
+subarray size that covers ALL HST data for the SkyCell regardless of the layer, then uses that to define the smallest
+WCS specification possible for the SkyCell.  The data from **p1889x07y19** actually only covers about 1/4 of the entire
+SkyCell resulting in a WCS defined as:
+
+.. code:: python
+
+    >>> wcs1889 = HSTWCS('hst_skycell-p1889x07y19_acs_wfc_f775w_all_drc.fits', ext=1)
+    >>> print(wcs1889)
+
+    WCS Keywords
+    Number of WCS axes: 2
+    CTYPE : 'RA---TAN'  'DEC--TAN'
+    CRVAL : 180.0  26.0
+    CRPIX : 94360.0  -171093.0
+    CD1_1 CD1_2  : -1.1111111111111e-05  0.0
+    CD2_1 CD2_2  : 0.0  1.11111111111111e-05
+    NAXIS : 10840  11672
+
+We can see how the exposures land in the SkyCell as defined with this subarray WCS.
+
+.. list-table::
+
+  * - .. figure:: images/skycell-p1889x07y19_f775w_full.jpg
+         :figwidth: 95%
+         :alt: SkyCell p1889x07y19 WFC3/UVIS F775W layer.
+
+         All the WFC3/UVIS F775W exposures that overlap SkyCell **p1889x07y19**.
+
+    -  .. figure:: images/skycell-p1889x07y19_f105w_full.jpg
+          :figwidth: 95%
+          :alt: SkyCell p1889x07y19 WFC3/IR F105W layer.
+
+          All the WFC3/IR F105W exposures that overlap SkyCell **p1889x07y19**.
+
+
+These figures demonstrate how the SkyCell subarray has been defined to cover only the exposures that overlap this
+SkyCell while minimizing the amount of empty space in these layers making these products as small as possible without
+resorting to compression.
+
+.. note::
+  These layers are defined as subarrays of the entire SkyCell WCS, which are just subarrays of the ProjectionCell.
+  As a result, any position in these layers refers to the exact same position on the sky
+  as defined in the SkyCell or ProjectionCell.  A source at (x,y)=(7936, 11133) will have the same sky coordinates
+  regardless of what SkyCell layer it was measured in, including an array defined for the entire SkyCell.
+
+
+File Format
+============
+The SkyCell layer mosaics get generated using AstroDrizzle using a custom set of rules for defining what
+keywords go into the primary and science extension headers. These files have the same set of extensions
+as all other drizzled products; namely, PRIMARY, SCI, WHT, CTX and HDRTAB.
+
+MVM-specific keywords
+---------------------
+MVM mosaics, by definition, include the contributions of many exposures (in most cases) potentially taken
+at many different times for a specific section of the sky.  Header keywords have been defined to provide
+some information on the unique characteristics of these mosaics and the contribution of the exposures to
+the mosaic.  This unique set of keywords defined in the SCI headers of MVM mosaics includes:
```

Comment:
Update from SCI header to Primary header.  Also include the MEANWHT and MEDWHT.

Reply:
Revised in [6b624e9](https://github.com/spacetelescope/drizzlepac/pull/1322/commits/6b624e920b7967f2d8c80485395ca1254dcb447b)